File: app/src/main/java/com/cyder/atsushi/youtubesync/json_data/SearchResponse.java
package com.cyder.atsushi.youtubesync.json_data;

/**
 * Created by chigichan24 on 2017/11/28.
 */

public class SearchResponse extends JsonParameter{
    public String next_page_token;
    public String prev_page_token;
    public String total_results;
    public String results_per_page;
    public Video[] items;
}

File: app/src/main/kotlin/com/cyder/atsushi/youtubesync/api/response/SearchResponse.kt
package com.cyder.atsushi.youtubesync.api.response

import com.google.gson.annotations.SerializedName

data class SearchResponse (
    @SerializedName("next_page_token") val nextPageToken: String?,
    @SerializedName("prev_page_token") val prevPageToken: String?,
    @SerializedName("total_results") val totalResults: Int,
    @SerializedName("results_per_page") val resultsPerPage : Int,
    val items: List<Video>
)